instrumentation-core/spring/starters/jaeger-exporter-starter/jaeger-exporter-starter.gradle

```diff
@@ -0,0 +1,19 @@
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
+  springbootVersion = "2.3.1.RELEASE"
+}
+
+group = 'io.opentelemetry.instrumentation'
+
+apply from: "$rootDir/gradle/java.gradle"
+apply from: "$rootDir/gradle/publish.gradle"
+
+sourceCompatibility = '8'
+
+dependencies {
+  api group: "org.springframework.boot", name: "spring-boot-starter", version: versions.springboot
+  api project(':instrumentation-core:spring:starters:spring-starter')
+  api deps.opentelemetryJaeger
+  implementation group: "io.grpc", name: "grpc-netty-shaded", version: "1.30.2"
```

Comment:
@mabdinur @anuraaga any thoughts on including `grpc-netty` or `grpc-netty-shaded` via the starters?

Reply:
Starters should work out of the box so this seems reasonable to me